README.md

```diff
@@ -100,6 +99,17 @@ hive.allow-drop-table=true
 hive.allow-rename-table=true
 ```
 
+#### Incremental models
+
+The incremental strategy currently supported by this adapter is to append new records
+without updating/overwriting any existing data from the target model.
+
+```
+{{
+    config(materialized = 'incremental')
+}}
```

Comment:
Is it required to specify `incremental_strategy='append'` here?

Reply:
No, not anymore. 
Since the current `incremental` functionality is only about appending, there's no more need to specify the "strategy"